lib/python/picongpu/plugins/plot_mpl/energy_histogram_visualizer.py

```diff
@@ -0,0 +1,142 @@
+"""
+This file is part of the PIConGPU.
+
+Copyright 2017-2018 PIConGPU contributors
+Authors: Sebastian Starke
+License: GPLv3+
+"""
+
+
+from picongpu.plugins.energy_histogram import EnergyHistogram
+import matplotlib.pyplot as plt
+
+
+class Visualizer:
+    """
+    Class for creation of histogram plots on a logscaled y-axis.
+    """
+
+    def __init__(self, run_directory):
+        """
+        Parameters
+        ----------
+        run_directory : string
+            path to the run directory of PIConGPU
+            (the path before ``simOutput/``)
+        """
+        if run_directory is None:
+            raise ValueError('The run_directory parameter can not be None!')
+
+        self.energy_histogram = EnergyHistogram(run_directory)
+        self.plt_obj = None
+
+    def visualize(self, iteration, ax, species='e', species_filter="all",
+                  include_overflow=False, **kwargs):
```

Comment:
I would remove `include_overflow` as an option since it will not be well visualized if it is included

Reply:
Would you completely remove it or be able to pass it down to the reader as kwargs? So would you want
```
counts, energy_bins = self.energy_histogram.get(
            species=species,
            species_filter=species_filter,
            iteration=iteration,
            **kwargs)
```
instead of
```
counts, energy_bins = self.energy_histogram.get(
            species=species,
            species_filter=species_filter,
            iteration=iteration,
            include_overflow=include_overflow)
```

Reply:
For this specific visualization, which is not great to investigate overflows, I would say we just completely remove it.
If it's easy to pass in kwargs and that's the right place I won't stop you from allowing it there :)